### PR TITLE
[DNM]Antimatter reactor update

### PR DESCRIPTION
**Pull request by silicons**
:cl:
tweak: Antimatter reactors and fuel now cost less! Also, they are not destroyed if you try to link them without a core!
/:cl:

### CODE REVIEW

File: code/modules/power/antimatter/shielding.dm [hunk 1]
```diff
@@ -218,6 +220,10 @@
 		qdel(src)
 	return
 
+/obj/machinery/am_shielding/proc/on_damage()
+	color = "#ff0000"
+	spawn(1)
```
Comment (RemieRichards) [inline code/modules/power/antimatter/shielding.dm]: no.

Comment (kevinz000) [inline code/modules/power/antimatter/shielding.dm]: rip

**Comment by kevinz000** [comment]
VG sprites are apparently the same and the interface is a no go because I can't into TGUI, so ready.

File: code/modules/power/antimatter/shielding.dm [hunk 2]
```diff
@@ -218,6 +220,12 @@
 		qdel(src)
 	return
 
+/obj/machinery/am_shielding/proc/on_damage()
+	color = "#ff0000"
```
Comment (ChangelingRain) [inline code/modules/power/antimatter/shielding.dm]: use the atom color procs!

**Comment by MrPerson** [comment]
Care to comment on #22653

**Comment by kevinz000** [comment]
@MrPerson worked absolutely perfectly for something that isn't tgui when I tested one in thunderdome.
I can't into tgui if I do the interface over I'll have to spend way more time before this is ready.

**Comment by optimumtact** [comment]
I have a better idea, lets remove this engine that will never be used and is barely functional and focus on the actually used engine the supermatter

**Comment by kevinz000** [comment]
@optimumtact the whole reason it's not used is because it was expensive and no one knows how to set it up, this is a useful engine.
DNM anyways because I need to add some more features like being harder to blow up.

**Comment by optimumtact** [comment]
it's not a useful engine, it's a non useful engine that will rarely if ever be used and should simply not exist Our design lead kor has already stated a desire publically to move to a single engine implementation and avoid the continual sprawl of engines

**Comment by Shadowlight213** [comment]
@optimumtact I've been discussing with kevinz, and we think that it would be a good higher tier engine for #24564 if it gets merged.
The amount of power that can be obtained is magnitudes greater than the sing or tesla.

**Comment by optimumtact** [comment]
The push is to one engine, not multiple tiers of engines @KorPhaeron

**Comment by MrPerson** [comment]
Producing even more power is a con, not a pro.

**Comment by Shadowlight213** [comment]
oranges explained his reasoning in coderbus, and I agree with them about only focusing on the sm for power generation.
If we keep the am, I think it would be better as some sort of system to make antimatter for use in am bombs or end tier science. More interdepartmental cooperation memes with science needing engineering to make enough power to make antimatter.

**Comment by KorPhaeron** [comment]
Agree with citrus, sorry

**Pull request closed without merging**